src/examples/Cluster/Metrics/Samples.Cluster.AdaptiveGroup/Samples.Cluster.AdaptiveGroup.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Should consume `common.props` here and specify the target framework using that file - like how we do here:

https://github.com/akkadotnet/akka.net/blob/b7630e5bf6e81631ace79bf3db0de38818a09f02/src/examples/Cluster/Samples.Cluster.Simple/Samples.Cluster.Simple.csproj#L3-L7

Reply:
Also - need to add `<IsPackable>false</IsPackable>` here so we don't accidentally create a NuGet package